src/types.ts

```diff
@@ -38,16 +38,24 @@ const asStatus = asValue(
   'refunded',
   'other'
 )
+
+const asSafeNumber = (raw: any): number => {
+  if (isNaN(raw) || raw === null) {
+    return 0
+  }
+  return raw
```

Comment:
You should return `asNumber(raw)` here. Otherwise strings & such would pass through the cleaner.

Reply:
good catch.